viewer/package.json

```diff
@@ -90,21 +91,23 @@
     "worker-plugin": "^5.0.0"
   },
   "dependencies": {
-    "@cognite/potree-core": "^1.1.6",
+    "@cognite/potree-core": "1.2.0",
     "@cognite/reveal-parser-worker": "1.1.1",
     "@cognite/sdk-core": "^2.1.2",
     "@tweenjs/tween.js": "^18.6.4",
+    "@types/jsdom": "^16.2.10",
     "@types/lodash": "^4.14.170",
     "@types/skmeans": "^0.11.0",
```

Comment:
actually whatever that doesn't leak into public API must not be in dependencies. I don't think we provide lodash API anywhere?